### PR TITLE
Update trappy and fix cpu_idle

### PR DESCRIPTION
**Pull request by bjackman**
Some people have been using a later trappy version than the one specified in the submodules. That version contains a commit by me that breaks LISA's idle analysis module, so here's a fix for it.

### CODE REVIEW

File: libs/utils/trace.py [hunk 1]
```diff
@@ -33,7 +33,6 @@
 from trappy.utils import listify
 
 
-NON_IDLE_STATE = 4294967295
```
Comment (mdigiorgio) [inline libs/utils/trace.py]: I'd still keep `NON_IDLE_STATE = -1`, it makes the code more readable.

Comment (bjackman) [inline libs/utils/trace.py]: Agreed.

**Comment by bjackman** [comment]
Pushed a new version w/ comment addressed

**Comment by mdigiorgio** [comment]
LGTM, let's merge it.

**Pull request merged**